Assets/RetroTVEffects/Resources/Shaders/ImageEffects/RetroTV.cginc

```diff
@@ -37,6 +37,7 @@
 // PARAMETERS
 // ====================
 sampler2D _MainTex;
+float4 _MainTex_ST;
```

Comment:
Thanks @radiatoryang. Just a note to users on version 2019.1.1f1, you will also need to change ```Graphics.DrawProceduralIndirect``` to ```Graphics.DrawProceduralIndirectNow``` in the ```DepthOfField.cs``` file.